Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2017-06-13
Initial Package Version: 20170524
Upstream Status: Applied
Origin: Upstream, r44495
Description: Fixes use of babel for English : the UKenglish and
USenglish files had obviously been created and tested on a
case-insensitive filesystem.

If you have already installed 2017 texlive, you can apply this
(as root, with /opt/texlive/2017/bin/*linux/ on your PATH) by
cd'ing to /opt/texlive/2017, applying it with patch -p1, and
then running 'mktexlsr' so that the updated files can be found.

diff -Naur texlive-20170524-texmf/texmf-dist/doc/generic/babel-english/README texlive-20170524-texmf-r44495/texmf-dist/doc/generic/babel-english/README
--- texlive-20170524-texmf/texmf-dist/doc/generic/babel-english/README	2017-01-24 15:32:30.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/doc/generic/babel-english/README	2017-06-13 02:42:09.427527728 +0100
@@ -1,11 +1,13 @@
-English 3.3q
+English 3.3r
 ------------
 
 This is the babel style for English.
 
 There are no new features in 3.3q, but "dialects" will be
-loaded with proxy files (american.ldf, british.ldf, etc.)
-with the forthcoming babel 3.10.
+loaded with proxy files (american.ldf, british.ldf, etc.).
+
+Version 3.3r just fixes the names for UKenglish.ldf and
+USenglish.ldf (files were created lowercased).
 
 ----------
-2017-01-24
\ No newline at end of file
+2017-06-06
\ No newline at end of file
diff -Naur texlive-20170524-texmf/texmf-dist/source/generic/babel-english/english.dtx texlive-20170524-texmf-r44495/texmf-dist/source/generic/babel-english/english.dtx
--- texlive-20170524-texmf/texmf-dist/source/generic/babel-english/english.dtx	2017-01-24 15:32:30.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/source/generic/babel-english/english.dtx	2017-06-13 02:42:09.427527728 +0100
@@ -42,7 +42,7 @@
 %<canadian>\ProvidesLanguage{canadian}
 %\fi
 %\ProvidesFile{english.dtx}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %\iffalse
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
diff -Naur texlive-20170524-texmf/texmf-dist/source/generic/babel-english/english.ins texlive-20170524-texmf-r44495/texmf-dist/source/generic/babel-english/english.ins
--- texlive-20170524-texmf/texmf-dist/source/generic/babel-english/english.ins	2017-01-24 15:32:30.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/source/generic/babel-english/english.ins	2017-06-13 02:42:09.427527728 +0100
@@ -32,7 +32,7 @@
 %%
 %% --------------- start of docstrip commands ------------------
 %%
-\def\filedate{2017/01/24}
+\def\filedate{2017/06/06}
 \def\batchfile{english.ins}
 \input docstrip.tex
 
@@ -133,9 +133,9 @@
 \usepreamble\mainpreamble
 \generate{\file{english.ldf}{\from{english.dtx}{english,code}}
           \file{american.ldf}{\from{english.dtx}{american}}
-          \file{usenglish.ldf}{\from{english.dtx}{usenglish}}
+          \file{USenglish.ldf}{\from{english.dtx}{usenglish}}
           \file{british.ldf}{\from{english.dtx}{british}}
-          \file{ukenglish.ldf}{\from{english.dtx}{ukenglish}}
+          \file{UKenglish.ldf}{\from{english.dtx}{ukenglish}}
           \file{australian.ldf}{\from{english.dtx}{australian}}
           \file{newzealand.ldf}{\from{english.dtx}{newzealand}}
           \file{canadian.ldf}{\from{english.dtx}{canadian}}
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/american.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/american.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/american.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/american.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{american}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/australian.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/australian.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/australian.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/australian.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{australian}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/british.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/british.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/british.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/british.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{british}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/canadian.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/canadian.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/canadian.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/canadian.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{canadian}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/english.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/english.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/english.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/english.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{english}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/newzealand.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/newzealand.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/newzealand.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/newzealand.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -38,7 +38,7 @@
 %% and covered by LPPL is defined by the unpacking scripts (with
 %% extension .ins) which are part of the distribution.
 \ProvidesLanguage{newzealand}
-        [2017/01/24 v3.3q English support from the babel system]
+        [2017/06/06 v3.3r English support from the babel system]
 %% File 'english.dtx'
 %% Babel package for LaTeX version 2e
 %% Copyright (C) 1989 - 2005
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/ukenglish.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/ukenglish.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/ukenglish.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/ukenglish.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -1,67 +0,0 @@
-%%
-%% This is file `ukenglish.ldf',
-%% generated with the docstrip utility.
-%%
-%% The original source files were:
-%%
-%% english.dtx  (with options: `ukenglish')
-%% This is a generated file.
-%% 
-%% Copyright 1989-2005 Johannes L. Braams and any individual authors
-%%           2013-2017 Javier Bezos, Johannes L. Braams and any individual authors
-%% listed elsewhere in this file.  All rights reserved.
-%% 
-%% This file was generated from file(s) of the Babel system.
-%% ---------------------------------------------------------
-%% 
-%% It may be distributed and/or modified under the
-%% conditions of the LaTeX Project Public License, either version 1.3
-%% of this license or (at your option) any later version.
-%% The latest version of this license is in
-%%   http://www.latex-project.org/lppl.txt
-%% and version 1.3 or later is part of all distributions of LaTeX
-%% version 2003/12/01 or later.
-%% 
-%% This work has the LPPL maintenance status "maintained".
-%% 
-%% The Current Maintainer of this work is Javier Bezos.
-%% 
-%% This file may only be distributed together with a copy of the Babel
-%% system. You may however distribute the Babel system without
-%% such generated files.
-%% 
-%% The list of all files belonging to the Babel distribution is
-%% given in the file `manifest.bbl'. See also `legal.bbl for additional
-%% information.
-%% 
-%% The list of derived (unpacked) files belonging to the distribution
-%% and covered by LPPL is defined by the unpacking scripts (with
-%% extension .ins) which are part of the distribution.
-\ProvidesLanguage{UKenglish}
-        [2017/01/24 v3.3q English support from the babel system]
-%% File 'english.dtx'
-%% Babel package for LaTeX version 2e
-%% Copyright (C) 1989 - 2005
-%%           by Johannes Braams, TeXniek
-%%           2013-2017 Javier Bezos, Johannes Braams
-\input english.ldf\relax
-%%
-%% \CharacterTable
-%%  {Upper-case    \A\B\C\D\E\F\G\H\I\J\K\L\M\N\O\P\Q\R\S\T\U\V\W\X\Y\Z
-%%   Lower-case    \a\b\c\d\e\f\g\h\i\j\k\l\m\n\o\p\q\r\s\t\u\v\w\x\y\z
-%%   Digits        \0\1\2\3\4\5\6\7\8\9
-%%   Exclamation   \!     Double quote  \"     Hash (number) \#
-%%   Dollar        \$     Percent       \%     Ampersand     \&
-%%   Acute accent  \'     Left paren    \(     Right paren   \)
-%%   Asterisk      \*     Plus          \+     Comma         \,
-%%   Minus         \-     Point         \.     Solidus       \/
-%%   Colon         \:     Semicolon     \;     Less than     \<
-%%   Equals        \=     Greater than  \>     Question mark \?
-%%   Commercial at \@     Left bracket  \[     Backslash     \\
-%%   Right bracket \]     Circumflex    \^     Underscore    \_
-%%   Grave accent  \`     Left brace    \{     Vertical bar  \|
-%%   Right brace   \}     Tilde         \~}
-%%
-\endinput
-%%
-%% End of file `ukenglish.ldf'.
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/UKenglish.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/UKenglish.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/UKenglish.ldf	1970-01-01 01:00:00.000000000 +0100
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/UKenglish.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -0,0 +1,67 @@
+%%
+%% This is file `UKenglish.ldf',
+%% generated with the docstrip utility.
+%%
+%% The original source files were:
+%%
+%% english.dtx  (with options: `ukenglish')
+%% This is a generated file.
+%% 
+%% Copyright 1989-2005 Johannes L. Braams and any individual authors
+%%           2013-2017 Javier Bezos, Johannes L. Braams and any individual authors
+%% listed elsewhere in this file.  All rights reserved.
+%% 
+%% This file was generated from file(s) of the Babel system.
+%% ---------------------------------------------------------
+%% 
+%% It may be distributed and/or modified under the
+%% conditions of the LaTeX Project Public License, either version 1.3
+%% of this license or (at your option) any later version.
+%% The latest version of this license is in
+%%   http://www.latex-project.org/lppl.txt
+%% and version 1.3 or later is part of all distributions of LaTeX
+%% version 2003/12/01 or later.
+%% 
+%% This work has the LPPL maintenance status "maintained".
+%% 
+%% The Current Maintainer of this work is Javier Bezos.
+%% 
+%% This file may only be distributed together with a copy of the Babel
+%% system. You may however distribute the Babel system without
+%% such generated files.
+%% 
+%% The list of all files belonging to the Babel distribution is
+%% given in the file `manifest.bbl'. See also `legal.bbl for additional
+%% information.
+%% 
+%% The list of derived (unpacked) files belonging to the distribution
+%% and covered by LPPL is defined by the unpacking scripts (with
+%% extension .ins) which are part of the distribution.
+\ProvidesLanguage{UKenglish}
+        [2017/06/06 v3.3r English support from the babel system]
+%% File 'english.dtx'
+%% Babel package for LaTeX version 2e
+%% Copyright (C) 1989 - 2005
+%%           by Johannes Braams, TeXniek
+%%           2013-2017 Javier Bezos, Johannes Braams
+\input english.ldf\relax
+%%
+%% \CharacterTable
+%%  {Upper-case    \A\B\C\D\E\F\G\H\I\J\K\L\M\N\O\P\Q\R\S\T\U\V\W\X\Y\Z
+%%   Lower-case    \a\b\c\d\e\f\g\h\i\j\k\l\m\n\o\p\q\r\s\t\u\v\w\x\y\z
+%%   Digits        \0\1\2\3\4\5\6\7\8\9
+%%   Exclamation   \!     Double quote  \"     Hash (number) \#
+%%   Dollar        \$     Percent       \%     Ampersand     \&
+%%   Acute accent  \'     Left paren    \(     Right paren   \)
+%%   Asterisk      \*     Plus          \+     Comma         \,
+%%   Minus         \-     Point         \.     Solidus       \/
+%%   Colon         \:     Semicolon     \;     Less than     \<
+%%   Equals        \=     Greater than  \>     Question mark \?
+%%   Commercial at \@     Left bracket  \[     Backslash     \\
+%%   Right bracket \]     Circumflex    \^     Underscore    \_
+%%   Grave accent  \`     Left brace    \{     Vertical bar  \|
+%%   Right brace   \}     Tilde         \~}
+%%
+\endinput
+%%
+%% End of file `UKenglish.ldf'.
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/usenglish.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/usenglish.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/usenglish.ldf	2017-01-24 21:38:15.000000000 +0000
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/usenglish.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -1,67 +0,0 @@
-%%
-%% This is file `usenglish.ldf',
-%% generated with the docstrip utility.
-%%
-%% The original source files were:
-%%
-%% english.dtx  (with options: `usenglish')
-%% This is a generated file.
-%% 
-%% Copyright 1989-2005 Johannes L. Braams and any individual authors
-%%           2013-2017 Javier Bezos, Johannes L. Braams and any individual authors
-%% listed elsewhere in this file.  All rights reserved.
-%% 
-%% This file was generated from file(s) of the Babel system.
-%% ---------------------------------------------------------
-%% 
-%% It may be distributed and/or modified under the
-%% conditions of the LaTeX Project Public License, either version 1.3
-%% of this license or (at your option) any later version.
-%% The latest version of this license is in
-%%   http://www.latex-project.org/lppl.txt
-%% and version 1.3 or later is part of all distributions of LaTeX
-%% version 2003/12/01 or later.
-%% 
-%% This work has the LPPL maintenance status "maintained".
-%% 
-%% The Current Maintainer of this work is Javier Bezos.
-%% 
-%% This file may only be distributed together with a copy of the Babel
-%% system. You may however distribute the Babel system without
-%% such generated files.
-%% 
-%% The list of all files belonging to the Babel distribution is
-%% given in the file `manifest.bbl'. See also `legal.bbl for additional
-%% information.
-%% 
-%% The list of derived (unpacked) files belonging to the distribution
-%% and covered by LPPL is defined by the unpacking scripts (with
-%% extension .ins) which are part of the distribution.
-\ProvidesLanguage{USenglish}
-        [2017/01/24 v3.3q English support from the babel system]
-%% File 'english.dtx'
-%% Babel package for LaTeX version 2e
-%% Copyright (C) 1989 - 2005
-%%           by Johannes Braams, TeXniek
-%%           2013-2017 Javier Bezos, Johannes Braams
-\input english.ldf\relax
-%%
-%% \CharacterTable
-%%  {Upper-case    \A\B\C\D\E\F\G\H\I\J\K\L\M\N\O\P\Q\R\S\T\U\V\W\X\Y\Z
-%%   Lower-case    \a\b\c\d\e\f\g\h\i\j\k\l\m\n\o\p\q\r\s\t\u\v\w\x\y\z
-%%   Digits        \0\1\2\3\4\5\6\7\8\9
-%%   Exclamation   \!     Double quote  \"     Hash (number) \#
-%%   Dollar        \$     Percent       \%     Ampersand     \&
-%%   Acute accent  \'     Left paren    \(     Right paren   \)
-%%   Asterisk      \*     Plus          \+     Comma         \,
-%%   Minus         \-     Point         \.     Solidus       \/
-%%   Colon         \:     Semicolon     \;     Less than     \<
-%%   Equals        \=     Greater than  \>     Question mark \?
-%%   Commercial at \@     Left bracket  \[     Backslash     \\
-%%   Right bracket \]     Circumflex    \^     Underscore    \_
-%%   Grave accent  \`     Left brace    \{     Vertical bar  \|
-%%   Right brace   \}     Tilde         \~}
-%%
-\endinput
-%%
-%% End of file `usenglish.ldf'.
diff -Naur texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/USenglish.ldf texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/USenglish.ldf
--- texlive-20170524-texmf/texmf-dist/tex/generic/babel-english/USenglish.ldf	1970-01-01 01:00:00.000000000 +0100
+++ texlive-20170524-texmf-r44495/texmf-dist/tex/generic/babel-english/USenglish.ldf	2017-06-13 02:42:09.427527728 +0100
@@ -0,0 +1,67 @@
+%%
+%% This is file `USenglish.ldf',
+%% generated with the docstrip utility.
+%%
+%% The original source files were:
+%%
+%% english.dtx  (with options: `usenglish')
+%% This is a generated file.
+%% 
+%% Copyright 1989-2005 Johannes L. Braams and any individual authors
+%%           2013-2017 Javier Bezos, Johannes L. Braams and any individual authors
+%% listed elsewhere in this file.  All rights reserved.
+%% 
+%% This file was generated from file(s) of the Babel system.
+%% ---------------------------------------------------------
+%% 
+%% It may be distributed and/or modified under the
+%% conditions of the LaTeX Project Public License, either version 1.3
+%% of this license or (at your option) any later version.
+%% The latest version of this license is in
+%%   http://www.latex-project.org/lppl.txt
+%% and version 1.3 or later is part of all distributions of LaTeX
+%% version 2003/12/01 or later.
+%% 
+%% This work has the LPPL maintenance status "maintained".
+%% 
+%% The Current Maintainer of this work is Javier Bezos.
+%% 
+%% This file may only be distributed together with a copy of the Babel
+%% system. You may however distribute the Babel system without
+%% such generated files.
+%% 
+%% The list of all files belonging to the Babel distribution is
+%% given in the file `manifest.bbl'. See also `legal.bbl for additional
+%% information.
+%% 
+%% The list of derived (unpacked) files belonging to the distribution
+%% and covered by LPPL is defined by the unpacking scripts (with
+%% extension .ins) which are part of the distribution.
+\ProvidesLanguage{USenglish}
+        [2017/06/06 v3.3r English support from the babel system]
+%% File 'english.dtx'
+%% Babel package for LaTeX version 2e
+%% Copyright (C) 1989 - 2005
+%%           by Johannes Braams, TeXniek
+%%           2013-2017 Javier Bezos, Johannes Braams
+\input english.ldf\relax
+%%
+%% \CharacterTable
+%%  {Upper-case    \A\B\C\D\E\F\G\H\I\J\K\L\M\N\O\P\Q\R\S\T\U\V\W\X\Y\Z
+%%   Lower-case    \a\b\c\d\e\f\g\h\i\j\k\l\m\n\o\p\q\r\s\t\u\v\w\x\y\z
+%%   Digits        \0\1\2\3\4\5\6\7\8\9
+%%   Exclamation   \!     Double quote  \"     Hash (number) \#
+%%   Dollar        \$     Percent       \%     Ampersand     \&
+%%   Acute accent  \'     Left paren    \(     Right paren   \)
+%%   Asterisk      \*     Plus          \+     Comma         \,
+%%   Minus         \-     Point         \.     Solidus       \/
+%%   Colon         \:     Semicolon     \;     Less than     \<
+%%   Equals        \=     Greater than  \>     Question mark \?
+%%   Commercial at \@     Left bracket  \[     Backslash     \\
+%%   Right bracket \]     Circumflex    \^     Underscore    \_
+%%   Grave accent  \`     Left brace    \{     Vertical bar  \|
+%%   Right brace   \}     Tilde         \~}
+%%
+\endinput
+%%
+%% End of file `USenglish.ldf'.
